R/modules.R

```diff
@@ -48,14 +48,12 @@ setOldClass("teal_modules")
 #' @param server_args (named `list`) with additional arguments passed on to the server function.
 #' @param ui_args (named `list`) with additional arguments passed on to the UI function.
 #' @param x (`teal_module` or `teal_modules`) Object to format/print.
-#' @param indent (`integer(1)`) Indention level; each nested element is indented one level more.
-#' @param transformers (`list` of `teal_data_module`) that will be applied to transform the data.
+#' @param transformers (`list` of `teal_transform_module`) that will be applied to transform module's data input.
```

Comment:
We need to rather link `transformers` with `decorators` and don't refer `teal_transform_module` to `teal_data_module`.